### DISCUSSION TITLE
Publish weekly meetups in some form?

### DISCUSSION THREAD

**Discussion by jaalibandar** [cables-gl/cables_docs#127] [Feature Requests] [answered]
Hello, I am a great fan of the weekly meetups but I miss out on them quite a lot as it gets pretty late for me here in India like 12PM - 2PM ... was wondering if there could be a way to check them out later - its very enriching to the overall cables community I feel - however I do understand that they might not always be polished enough / a bit too informal for youtube - but if they could be made accessible to interested community members in some form that would be really awesome :)

**Comment by steam0r** [comment, marked as answer]
hey,

we have no plans on recording the meetups, since that would mean that we need request of all participants (that can always be revoked at any time) and would also kill the informal nature of the meetup with everyone doing show-and-tell. we tried this in the past with twitch streams but these were too one-directional.

we try to pick up topics from the meetups for video tutorials though, so that might help...
